app/assets/stylesheets/application.scss

```diff
@@ -1,10 +1,80 @@
-@import "primer-css/index.scss";
+@import "primer-core/index.scss";
+@import "primer-marketing/index.scss";
 @import "addtocalendar";
 
+
+.alt-h1 {
+  line-height: 1.25;
```

Comment:
Instead of adding custom CSS to this style, you can use a utility class within the markup. This will help you have more flexibility with your classes. https://github.com/styleguide/css/styles/core/utilities/typography#line-height-styles (for this you'd use `.lh-condensed`